html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html

```diff
@@ -31,6 +31,10 @@
   test_computed_style_aspect_ratio("input", {type: "submit", width: width, height: height}, "auto");
 }
 
+function assert_cs(img, val) {
```

Comment:
Since there's just a single call to this I would inline it at the call site I think. (Or if it was used a lot, probably name it `assert_aspect_ratio` or something.)